search_data.json

```diff
@@ -0,0 +1,50 @@
+---
+layout: null
+---
+{
+  {% for item_hash in data.resource %}
+    {% assign object = item_hash[1] %}
+    {% assign id = item_hash[0] %}
+    {% assign collection_id = object.uri | split:'/' | last %}
+    {% assign subjects = "" | split: ',' %}
+    {% assign agents = "" | split: ',' %}
+    {% assign dates = "" | split: ',' %}
```

Comment:
You're creating empty list variables here, then pushing into them below, right? Just want to confirm my understanding of what's going on here.

Reply:
Yeah, that's what's happening. I figured we just needed a list of the names for subjects and agents without the other info, but I could be wrong.